### PR TITLE
Enable elementwise_div operator test=develop

### PR DESCRIPTION
**Pull request by mozga-intel**
Enable the elementwise div operator for the nGraph bridge. BERT model uses it.

### CODE REVIEW

**Review by baojun-nervana — DISMISSED** [review]
LGTM

File: paddle/fluid/operators/ngraph/ops/elementwise_div_op.h [hunk 1]
```diff
@@ -0,0 +1,103 @@
+/*Copyright (c) 2018 PaddlePaddle Authors. All Rights Reserved.
```
Comment (tensor-tang) [inline paddle/fluid/operators/ngraph/ops/elementwise_div_op.h]: 2019

File: python/paddle/fluid/tests/unittests/ngraph/test_elementwise_div_ngraph_op.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
+#	Copyright (c) 2018 PaddlePaddle Authors. All Rights Reserved.
```
Comment (tensor-tang) [inline python/paddle/fluid/tests/unittests/ngraph/test_elementwise_div_ngraph_op.py]: same above

File: paddle/fluid/operators/ngraph/ops/elementwise_div_op.h (hunk 1 above)
Comment (mozga-intel) [reply paddle/fluid/operators/ngraph/ops/elementwise_div_op.h]: Done.

File: python/paddle/fluid/tests/unittests/ngraph/test_elementwise_div_ngraph_op.py (hunk 2 above)
Comment (mozga-intel) [reply python/paddle/fluid/tests/unittests/ngraph/test_elementwise_div_ngraph_op.py]: Done.

**Comment by LeoZhao-BJ** [comment]
Conflicting files
paddle/fluid/operators/ngraph/ops/elementwise_node.h

this should be solved by syncing with latest code

**Comment by mozga-intel** [comment]
@tensor-tang done.

File: python/paddle/fluid/tests/unittests/ngraph/test_elementwise_div_ngraph_op.py [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import print_function
+
+import unittest, sys
+sys.path.append("../")
```
Comment (tensor-tang) [inline python/paddle/fluid/tests/unittests/ngraph/test_elementwise_div_ngraph_op.py:18]: why do this?

File: paddle/fluid/operators/ngraph/ops/elementwise_div_op.h [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+    auto dx = std::make_shared<ngraph::op::Divide>(dout, y_broadcast_reshape);
+
+    paddle::platform::SetOutputNode(op, "X@GRAD", dx, ngb_node_map);
+  }
+}
+}  // namespace ngraphs
+}  // namespace operators
+}  // namespace paddle
+
+REGISTER_NG_OP(elementwise_div_grad, BuildElementwiseDivGradNode);
```
Comment (tensor-tang) [inline paddle/fluid/operators/ngraph/ops/elementwise_div_op.h:103]: only need backward? why forward in another file?

File: python/paddle/fluid/tests/unittests/ngraph/test_elementwise_div_ngraph_op.py (hunk 3 above)
Comment (baojun-nervana) [reply python/paddle/fluid/tests/unittests/ngraph/test_elementwise_div_ngraph_op.py:18]: This is for python3. Python3 changes the behavior of relative import. Without this, it won't be able to locate the import path.

File: paddle/fluid/operators/ngraph/ops/elementwise_div_op.h (hunk 4 above)
Comment (baojun-nervana) [reply paddle/fluid/operators/ngraph/ops/elementwise_div_op.h:103]: the elementwise div is simpler and can be supported by the BuildElementwiseBinaryNode template.
https://github.com/PaddlePaddle/Paddle/blob/c8f325b65195c9eaf4c4f8c12fda9d2a7b0e9174/paddle/fluid/operators/ngraph/ops/elementwise_node.h#L77

**Review by tensor-tang — APPROVED** [review]

**Pull request merged**